cmake_templates/ZeekConfig.cmake.in

```diff
@@ -26,6 +26,12 @@ if ( MSVC )
     find_dependency(c-ares)
 endif ()
 
+set(_sanitizer_flags "@ZEEK_SANITIZER_CXX_FLAGS@")
+if ( _sanitizer_flags )
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${_sanitizer_flags}" PARENT_SCOPE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${_sanitizer_flags}" PARENT_SCOPE)
+endif ()
```

Comment:
Hmm, hmm, we have `ZeekPluginBootstrap.cmake.in` which feels and reads like that might be the better place if this is only about plugins?

And I suspect that then needs some updates in `cmake/ZeekPlugin.cmake` to recognize that variable then instead of the `PARENT_SCOPE` trick? We do something similar with `CMAKE_BUILD_TYPE` already.

That said, @Neverlord might have another thought/idea/opinion?